shared/js/ui/models/site-company-list.es6.js

```diff
@@ -17,18 +18,14 @@ SiteCompanyList.prototype = window.$.extend({},
 
         fetchAsyncData: function () {
```

Comment:
Optional: Would be nice to tidy this function up a little, e.g. using async/await, the new [method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) syntax and [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). Maybe out of scope though.

Reply:
The whole file is going in the bin soon anyway tbh. Let's limit diff noise but I take your point I've been making these kinda changes where possible.

Reply:
OK fair enough.